tests/src/roomHandlers/exampleRoom3Handler.test.ts

```diff
@@ -64,6 +66,7 @@ testSingletonModule(
           startingRoomState: {
             laps: 2,
           },
+          stateAssertionDescription: 'increments the lap count',
```

Comment:
Where does `increments the lap count` come from? It looks like if I change this value to any string it will pass. It will even pass if I change this to `null`. 🤔

Reply:
It's just a description for the test. The test helper can't know what it means for the room state properties to change, so I made it where you can say what it means. It gets passed to the `.assert` chained to the `testScenario` call within the helper.